tests/integration/link/test_title_prop.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import dash
+from dash.dependencies import Input, Output
+import dash_core_components as dcc
+import dash_html_components as html
+
+
+@pytest.mark.DCC768
+def test_liti001_prop(dash_dcc):
+    app = dash.Dash(__name__)
+    app.layout = html.Div(
+        [
+            dcc.Link("page 1", id="link1", href="/page-1", title="This is a test title!"),
+            dcc.Location(id="url", refresh=False),
+            html.Div(id="content")
+        ]
+    )
+    @app.callback(Output("content", "children"), [Input("link1", "title")])
+    def display_title(title):
+        return title
+
+    dash_dcc.start_server(app)
+
+    title_exists = dash_dcc.driver.execute_script(
+        '''
+        return document.getElementById("link1").getAttribute("title");
+        '''
+    )
```

Comment:
Does this work instead?
```
dash_dcc.find_element('#link1').get_attribute('title')
```

Reply:
Added in f80f844. This works, thank you for the suggestion.